node/src/debian/DEBIAN/postinst.sh

```diff
@@ -0,0 +1,21 @@
+#! /usr/bin/env bash
+
+set -e
+
+USERNAME="rnode"
+DIRECTORY="/var/lib/${USERNAME}"
+
+if id -u ${USERNAME} >/dev/null 2>&1; then
+    echo "User ${USERNAME} already exists."
+else
+    adduser --no-create-home --home /nonexistent --group --system ${USERNAME}
```

Comment:
Looking pretty good. Not a big deal but I might consider using useradd over adduser i.e. 
"useradd --system rnode"
or probably better
"useradd --system rnode --home-dir /var/lib/rnode --shell /bin/false" 
if you wanted to explicitly define home directory and shell to false, even though it doesn't populate anything from skeleton/template. I get the idea of using /nonexistent for security but I would probably specify rnode directory. I'm not sure /nonexistent really buys you any security as any good script would crawl the /var/lib dir.
 
useradd will only do what you tell it to do but adduser makes more default assumptions, like creating home directory. adduser is nice when adding users with name, contact info, home directory etc. but for system creation may not be the best choice unless called for.

So, even though they do the same thing it is a little less syntax with useradd and less chance of directory getting created on missing params.
reference link below for more if you haven't read it already.
https://askubuntu.com/questions/345974/what-is-the-difference-between-adduser-and-useradd
I'm sure there are many more vs posts out there. You might have more insight than I have. Those were just some thoughts.

Reply:
@jeremybusk - is it common practice to use `/var/lib/<whatever>` as home in cases like these?

Seems like there could be some unintended side effects if the user spawned a shell as rnode.  I'd sorta like to keep the contents of that directory clean and well-accounted for...

Reply:
@Dyonishw - let's stick with `adduser` for now and no `HOME` directory - we can refine this in the future.